src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier supplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < 3; i++) {
+            figures[i] = supplier.getRandomFigure();
+        }
+
+        figures[3] = new Circle("circle", "GREEN", 3);
+        figures[4] = new Square("square", "YELLOW", 2);
+        figures[5] = new RightTriangle("rightTriangle", "BLACK", 3, 6);
+
```

Comment:
```suggestion
```

you don't need this, you should create only default and random figures, using FigureSupplier